src/unitxt/api.py

```diff
@@ -92,7 +92,7 @@ def evaluate(predictions, data) -> List[Dict[str, Any]]:
 
 
 def post_process(predictions, data) -> List[Dict[str, Any]]:
```

Comment:
what is this function aim? It's not clear what it does and how the change reflect it

Reply:
is it only for inference? What does it means?

Reply:
This function is for the api to use only post processing on a data without the rest of the pipeline.